movies/src/components/movieDeletePopup/movieDeletePopup.js

```diff
@@ -1,11 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Modal from 'react-modal';
 import PropTypes from 'prop-types';
 import CloseButton from '../closeButton/CloseButton';
 import Button from '../common/button/Button';
 
 export default function MovieDeletePopup(props) {
   const { handleClose, isOpen } = props;
+
+  useEffect(()=> handleClose, [isOpen] );
+
   return (
     <Modal
```

Comment:
when modal is opened, the following warning appears: 
![image](https://user-images.githubusercontent.com/16895020/94664423-87a60b00-0313-11eb-942f-058352afa30d.png)

Reply:
Fixed